README.md

```diff
@@ -159,6 +159,28 @@ It is also possible to determine whether a layer supports a specific ApiType:
   };
 ```
 
+Gain and gamma effects are supported only when the blob is retrieved, because the effects are applied in the library. 
```

Comment:
The way the effects are applied is a good approach and it was quite a lot of work, but - I don't think users care about that... :) It is (and should be) transparent to them. For this reason, I would suggest simply stating:
```
Effects `gain` and `gamma` can also be applied:
```

Reply:
I didn't know how detailed the description about gain and gamma being applied by the library should be. 
Thanks, this looks better.

Reply:
Resolving so I can merge